themes/website/static/js/osdetect.js

```diff
@@ -183,28 +197,60 @@ if (window.navigator.userAgent.indexOf("Windows Phone") != -1) {
 	All the magic stuff the makes the buttons change and stuff
 */
 
+// Set platform header
+document.getElementById("downloadHeader").innerHTML = "Download Tox for " + OSName;
+
 // Set platform toxlock
 document.getElementById("platImg").src = "theme/img/plat/" + OSName.toLowerCase() + ".svg";
 
 // Remove normal button
 document.getElementById("defaultButton").innerHTML = "";
 
+var buttonArea = document.getElementById("buttonArea");
+
+
 // Loop through all links and make buttons and info modals
-for (var i = 0; i < clients.length; i++) {
+for (var i = 0; i < clients.length; i++) 
```

Comment:
Rewrite it with length being cached in the loop please (best performances in case the clients list grow)
```js
for (var i = 0, length = clients.length; i < length; i++) {
  ...
}
```

Reply:
Done.